packages/flutter/lib/src/material/app.dart

```diff
@@ -163,9 +166,45 @@ class MaterialApp extends StatefulWidget {
   /// This value is passed unmodified to [WidgetsApp.onGenerateTitle].
   final GenerateAppTitle onGenerateTitle;
 
-  /// The colors to use for the application's widgets.
+  /// Default visual properties, like colors fonts and shapes, for this app's
+  /// material widgets.
+  ///
+  /// A second [darkTheme] [ThemeData] value, which is used when the underlying
+  /// platform requests a "dark mode" UI, can also be specified.
+  ///
+  /// The default value of this property is a light [ThemeData.fallback].
+  ///
+  /// See also:
+  ///
+  /// * [MediaQueryData.platformBrightness], which indicates the platform's
+  ///    desired brightness and is used to automatically toggle between [theme]
+  ///    and [darkTheme] in [MaterialApp].
+  /// * [ThemeData.brightness], which indicates the [Brightness] of a theme's
+  ///    colors.
   final ThemeData theme;
 
+  /// The [ThemeData] to use when the platform specifically requests a dark
+  /// themed UI.
+  ///
+  /// Host platforms such as Android Pie can request a system-wide "dark mode"
+  /// when entering battery saver mode.
+  ///
+  /// When the host platform requests a [Brightness.dark] mode, you may want to
+  /// supply a [ThemeData.brightness] that's also [Brightness.dark].
+  ///
+  /// Uses [theme] instead when null. Defaults to a light [ThemeData.fallback]
+  /// theme when both [darkTheme] and [theme] are null.
+  ///
+  /// See also:
+  ///
+  /// * [MediaQueryData.platformBrightness], which indicates the platform's
```

Comment:
See also formatting is:
```
///  * foo...
///    bar
```

Two spaces after the `///`; four spaces after the `///` for the second and subsequent lines.

Reply:
Added a space to each "*" line in these two docs.